docs/MachineConfigDaemon.md

```diff
@@ -157,12 +157,16 @@ The updated list of optimized updates and behaviour (as of Openshift 4.8) is as
 1. [SSH Keys](./Update-SSHKeys.md) (updating ignition/passwd/users/sshAuthorizedKeys section in a MachineConfig)
 2. kube-apiserver-to-kubelet-signer CA cert (located at `/etc/kubernetes/kubelet-ca.crt`, 1 year expiry autorotated by the openshift-kubeapiserver operator)
 3. [Pull Secret](./PullSecret.md) (cluster-wide, located at `/var/lib/kubelet/config.json`).
+4. **Selected registries.conf changes(/etc/containers/registries.conf, e.g. ICSP changes)** It can only include following changes in registries.conf file:
```

Comment:
Maybe something like: `Selected registries.conf changes(/etc/containers/registries.conf, e.g. ICSP changes)` -> `Select registries.conf changes(/etc/containers/registries.conf. This file is generally changed via ICSP object changes. Only the following changes will cause no-reboot upgrades`

Reply:
fixed